modules/nf-core/seqtk/sample/meta.yml

```diff
@@ -0,0 +1,52 @@
+name: seqtk_sample
+description: Subsample reads from FASTQ files
+keywords:
+  - sample
+  - fastx
+  - reads
+tools:
+  - seqtk:
+      description: Seqtk is a fast and lightweight tool for processing sequences in
+        the FASTA or FASTQ format. Seqtk sample command subsamples sequences.
+      homepage: https://github.com/lh3/seqtk
+      documentation: https://docs.csc.fi/apps/seqtk/
```

Comment:
I actually think the README in the github repo is the official docs: https://github.com/lh3/seqtk#readme . The CSC page doesn't seem to add very much, and might not be stable.

Reply:
Fair enough, I'll change!